### DISCUSSION TITLE
is it collection navigation didnt work anymore ?

### DISCUSSION THREAD

**Discussion by iqraaaswad** [directus/directus#9204] [Archive]
![image](https://user-images.githubusercontent.com/58161890/139113701-f2128497-2224-4e84-a30d-354b424381f7.png)
i've configuring "collection navigation" at "Role and Permission" but i didnt see anything happened at Collections.
is it bug or i missing something ? 
directus version : Directus 9.0.0-rc.98

**Comment by rijkvanzanten** [comment]
See https://github.com/directus/directus/issues/8920

**Reply by iqraaaswad** [reply]
okay.. thanks
